website/views.py

```diff
@@ -68,3 +71,25 @@ def create_team():
         return render_template('create_team.html', user=current_user, athletes = Athlete.query.all(), coaches = Coach.query.all())
         
     return render_template('create_team.html', user=current_user, athletes = Athlete.query.all(), coaches = Coach.query.all())
+
+
+#coach Dasboard page
+@views.route('/teams/<string:id>', methods = ['GET', 'POST'])
+def coach_dashboard(id):
+    coach = Coach.query.filter_by(colby_id=current_user.colby_id).first()
+    currentTeam = Team.query.get(id)
```

Comment:
I feel like we don't need teamList since we can just access it by doing coach.teams